decentraland/unity-renderer
# feat: cursor unlocking with right click

## What does this PR change?

Allows you to unlock the cursor with the right click showing a hint in the middle of the screen for a couple of seconds whenever you lock it. The hint is only shown one time per session to avoid spamming the user.
Camera flick has been fixed when you unlock via this new input.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/cursor-unlocking-right-click
2. Once you are in world click on the screen to lock the cursor
3. A hint should be shown indicating that you can unlock it with with right click
4. The hint should have a duration of 3 seconds
5. Use the right click, the cursor should be unlocked
6. Lock the cursor again by clicking the screen, the hint should not appear again
7. Refresh the tab to start a new session and repeat from step 2

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
